inc/search.class.php

```diff
@@ -2977,8 +3114,10 @@ static function displaySearchoptionValue($request = []) {
     * @param string  $val            value search
     *
     * @return select string
+    *
+    * @since 10.0.0 Method is no longer static
    **/
-   static function addHaving($LINK, $NOT, $itemtype, $ID, $searchtype, $val) {
+   public function addHaving($LINK, $NOT, $itemtype, $ID, $searchtype, $val) {
```

Comment:
This method is used in some plugins (`fusioninventory` for instance), change should be listed in changelog.